src/components/openSourceDep/DashboardOpenSourceDependencies.vue

```diff
@@ -79,16 +81,156 @@
             </stats-card>
           </b-col>
       </b-row>
+      <!-- Language Counts -->
+      <b-row>
+        <b-col>
+          <stats-card>
+            <div slot="header" class="icon-success">
+              <b-icon icon="arrow-right" class="con-big text-center" style="color: green;"></b-icon>
+            </div>
+            <div slot="content" class="numbers">
+              <p>Node Count</p>
+              {{ $store.getters.vulnerabilities.metrics.nodeCount }}
+            </div>
+          </stats-card>
+        </b-col>
+        <b-col>
+          <stats-card>
+            <div slot="header" class="icon-success">
+              <b-icon icon="arrow-right" class="con-big text-center" style="color: green;"></b-icon>
```

Comment:
It would be great to get the Javascript logo from the public folder here (might need to add it first)